webpack/assets/javascripts/react_app/components/PF4/TableIndexPage/Table/Table.js

```diff
@@ -226,7 +226,7 @@ export const Table = ({
               }))}
         </Tbody>
       </PFTable>
-      {results.length > 0 && !errorMessage && !emptyMessage && bottomPagination}
```

Comment:
Could you please explain this change? I think pagination shouldnt be visible if the table has an empty state? or do you think we dont need to check for the message and just check that there are no results ?

Reply:
`emptyMessage` should not be the flag as to whether there's an absence of empty state; rather, the `results.length > 0` should suffice. With the `!emptyMessage` here, bottom pagination was absent from the table in https://github.com/Katello/katello/pull/11547/files